runtime/tests/runtime_integration_tests/test_boot.rs

```diff
@@ -80,6 +80,60 @@ fn test_update() {
     assert_eq!(fw_rev[1], 0xaabbccdd);
 }
 
+///This test will be enabled only on nightly runs depending on the presence of slow_tests feature
+#[cfg_attr(not(feature = "slow_tests"), ignore)]
+#[test]
+fn test_stress_update() {
+    let app_versions = [0xaaabbbbc, 0xaaabbbbd];
+    let image_options_0 = ImageOptions {
+        app_version: app_versions[0],
+        ..Default::default()
+    };
+    let image_options_1 = ImageOptions {
+        app_version: app_versions[1],
+        ..Default::default()
+    };
+
+    let image = [
+        caliptra_builder::build_and_sign_image(&FMC_WITH_UART, &APP_WITH_UART, image_options_0)
+            .unwrap()
+            .to_bytes()
+            .unwrap(),
+        caliptra_builder::build_and_sign_image(&FMC_WITH_UART, &APP_WITH_UART, image_options_1)
+            .unwrap()
+            .to_bytes()
+            .unwrap(),
+    ];
+
+    let mut model = run_rt_test(None, None, None);
+
+    let stress_num: u32 = 500;
```

Comment:
Instead of fully disabling the tests based on feature, can you do something like https://github.com/chipsalliance/caliptra-sw/blob/4cc5df2e6a286200687843d5d88253ba54efb800/rom/dev/tests/rom_integration_tests/test_idevid_derivation.rs#L82-L86

This makes sure the test gets run in the presubmits too instead of just nightly to prevent breakages.

Reply:
Updated the test to be run atleast once when the feature is disabled.